eng/dcppack/dcppack.csproj

```diff
@@ -39,8 +39,16 @@
 
   <Target Name="Build" />
 
+  <Target Name="AddPackageFiles" BeforeTargets="_GetPackageFiles">
```

Comment:
Same comment here, if we could instead hook up to one of the "public" targets: https://learn.microsoft.com/en-us/nuget/reference/msbuild-targets#advanced-extension-points-to-create-customized-package